AwsCryptographyPrimitives/README.md

```diff
@@ -1,12 +1,13 @@
-# AWS Cryptographic Internal AwsCryptographyPrimitives 
+# AWS Cryptographic Internal AwsCryptographyPrimitives
 
 The AWS Cryptographic Internal AwsCryptographyPrimitives is a modeled library used in the AWS Cryptographic Material Providers Library. This internal library makes no guarantees that functionality will be added or removed between minor versions.
 
-**DO NOT** take a standalone dependency on this library. 
+**DO NOT** take a standalone dependency on this library.
```

Comment:
Relief: I see what is happening.
Apparently, `*` is vague or overloaded in Markdown.
Prettier is replacing `*` used for bold with `_` for italics,
only when  it is `*` and not `**`.